internal/catalog/internal/controllers/workloadhealth/controller_test.go

```diff
@@ -500,15 +503,22 @@ func (suite *workloadHealthControllerTestSuite) TestController() {
 	// create a node to link things with
 	node := suite.injectNodeWithStatus("test-node", pbcatalog.Health_HEALTH_PASSING)
 
+	owner := resourcetest.ResourceID(&pbresource.ID{
+		Name: "test-identity",
+		Type: pbauth.WorkloadIdentityType,
+	}).
+		Write(suite.T(), suite.client)
+
 	// create the workload
 	workload := resourcetest.Resource(pbcatalog.WorkloadType, "test-workload").
 		WithData(suite.T(), workloadData(node.Id.Name)).
+		WithOwner(owner.Id).
```

Comment:
Why is a workload owned by workload identity?

Reply:
this is an unnecessary remnant.  removed in ad35daaa544d8b070bb03862f8b51e3f62d64df2